packages/lib/src/components/Table/TableControl/ResultCountLabel/index.tsx

```diff
@@ -14,6 +14,8 @@ export const ResultCountLabel: React.FC<Props> = ({
   label,
   numerator,
 }) => (
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore: FIXME. This component is throwing an invalid type
```

Comment:
same q, it has a red underline but I don't think it throws compile errors
```suggestion
```

Reply:
it throws compile errors when used across apps